ovs-p4rt/sidecar/testing/ipv4_tunnel_test.h

```diff
@@ -17,7 +17,7 @@ constexpr char IPV4_DST_ADDR[] = "192.168.17.5";
 constexpr int IPV4_PREFIX_LEN = 24;
 
 constexpr uint16_t SRC_PORT = 0x1066;
-constexpr uint16_t DST_PORT = 0x1984;
+constexpr uint16_t DST_PORT = 0x4224;
```

Comment:
0x1984 overflows when you swap the upper and lower bytes (0x8419) and multiply by two. Replaced it with a value that can be doubled without overflowing.